pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -251,10 +270,45 @@ func GatherContainerImages(i *Gatherer) func() ([]record.Record, []error) {
 			continueValue = pods.Continue
 		}
 
+		// Transform map into a list for sorting.
+		type tmpImageCount struct {
+			Image      string
+			StartMonth string
+			Count      int
+		}
+		imageCounts := []tmpImageCount{}
+		for img, countMap := range img2month2count {
+			for month, count := range countMap {
```

Comment:
Because we want to count all images by ID, shouldn't the Count be Sum of all counts from all months ?

Reply:
If I do that, it will become even more difficult to split that information apart after the sorting is done and the temporary structure will become quite messy (name, total count and a dictionary of month:count, from the top of my head). I guess it's true, but neither way is optimal. Either way, we should put much more emphasis on the old images, but there isn't any nice way of doing it.